data/ka'het/ka'het outfits.txt

```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although it is energy intensive, it is very effective; capable of single- handedly cooling some Het ships."
```

Comment:
1. "it is" is now used twice in this same sentence
2. changing the comma to a semicolon is incorrect
3. It's "single-handedy" (no space after the dash)

Reply:
Changed again, how about this? 
"This is the main cooler used by the Ka'het. Although energy intensive, it is very effective, capable of single-handedly cooling some Het by itself."

Reply:
That seems good to me.